src/rgw/rgw_cache.cc

```diff
@@ -25,24 +21,25 @@ class RGWGetObj_CB;
 
 int ObjectCache::get(const string& name, ObjectCacheInfo& info, uint32_t mask, rgw_cache_entry_info *cache_info)
 {
-  RWLock::RLocker l(lock);
 
+  std::shared_lock rl{lock};
```

Comment:
ah, ok, now they are back

Reply:
apologies, results of merge/rebase conflict artifacts